README.md

```diff
@@ -37,7 +43,24 @@ Requires a configuration file `configuration.json` with information for each eng
 ```
 If an `executablePath` or `parameterExpression` is omitted, the engine is expected to be hosted externally. An example of this is the `External` engine in the above configuration. Engines can optionally be marked `verbose` to print failed queries while the tests are run from [Run Tests for Engine](#run-tests-for-engine)
 
-##Run Tests for Engine
+`testTimeout`, `testCount`, `testSorting`, `queryComplexity` are all optional configurations.
+`testTimeout` sets the time limit in seconds for the duration of a test (default=30).
+`testCount` limits the number of tests to execute (default=all).
+`testSorting` determines how to sort the tests if `testCount` is set. There are four different sortings:
+* `Random` (default) - Takes `testCount` generated tests randomly
+* `FILO` - Takes the last `testCount` generated tests
+* `FIFO` - Takes the first `testCount` generated tests
+* `Fair` - Takes an equal amount of each test-sort, summing up to no more than `testCount` tests
```

Comment:
I believe that both FILO and FIFO are fair. Maybe "Fair" should be renamed to "Split"?

Reply:
Also I think it should be mentioned that Fair uses RoundRobin.